sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/review_checker_clients_v1/query.sql

```diff
@@ -0,0 +1,107 @@
+WITH shopping_metrics AS (
+  SELECT
+    client_info.client_id AS client_id,
+    DATE(submission_timestamp) AS submission_date,
+    SUM(
+      CASE
+        WHEN metrics.counter.shopping_product_page_visits IS NOT NULL
+          THEN COALESCE(CAST(metrics.counter.shopping_product_page_visits AS INT64), 0)
+        ELSE 0
+      END
+    ) AS shopping_product_page_visits,
+    CASE
+      WHEN metrics.boolean.shopping_settings_user_has_onboarded = TRUE
+        AND metrics.boolean.shopping_settings_component_opted_out = FALSE
+        THEN 1
+      ELSE 0
+    END AS is_opt_in,
```

Comment:
What's the reason for storing this as 1 or 0 rather than as a boolean?

Reply:
No specific reason. Please let me know if one is preferred over the other. However, the desktop already has them numeric so changing that could be cumbersome now.